todo-list/src/shims-tsx.d.ts

```diff
@@ -0,0 +1,13 @@
+import Vue, { VNode } from 'vue'
+
+declare global {
+  namespace JSX {
```

Comment:
What does that do? :thinking:

Reply:
we don't know either, that was also autogenerated. Probably also just autogenerated types for typescript